### DISCUSSION TITLE
Blander 4.0.1 update keeps failing with HTTP error 403. Forbidden

### DISCUSSION THREAD

**Discussion by oxoocoffee** [Homebrew/discussions#4923] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.21-22-gaef456f
ORIGIN: https://github.com/Homebrew/brew
HEAD: aef456fd03c143f2816617b9a0b459f375a3d9bf
Last commit: 6 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 2ba3cfb8295f8dcbc6429102581b712f49aa0c50
Core tap last commit: 41 minutes ago
Core tap branch: master
Core tap JSON: 22 Nov 23:08 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0.0
HOMEBREW_EDITOR: vi
HOMEBREW_MAKE_JOBS: 10
HOMEBREW_NO_AUTO_UPDATE: set
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.43.0 => /opt/homebrew/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 14.1.1-arm64
CLT: 15.0.0.0.1.1694021235
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: The filesystem on /System/Volumes/Data appears to be case-sensitive.
The default macOS filesystem is case-insensitive. Please report any apparent problems.
```

### Description of issue

First, I installed installed latest available Blander 3.x from the cask about a week before 4.0 was available. When 4.0 was available I upgraded to it just fine using `brew upgrade --cask blender`
Since 4.0.1 just came out, I tried to upgrade the same way. But it keeps failing to download.

It looks like `wget` is failing with an HTTP 403 error.

```bash
$ brew upgrade --cask blender blender 
==> Upgrading 1 outdated package:
blender 4.0.0 -> 4.0.1
==> Upgrading blender
==> Downloading https://raw.githubusercontent.com/Homebrew/homebrew-cask/f20d1ee011991e33c3cf564977aab8bd644f306c/Casks/b/blender.rb
Already downloaded: /Users/oxoocoffee/Library/Caches/Homebrew/downloads/46e693cdb45c2e6c98b851b6bff38f4342506a0fe09033ba8c4cd6998fba13c3--blender.rb
==> Downloading https://download.blender.org/release/Blender4.0/blender-4.0.1-macos-arm64.dmg
curl: (22) The requested URL returned error: 403
```

If I type a direct URL to Safari `https://download.blender.org/release/Blender4.0/blender-4.0.1-macos-arm64.dmg` I can download the the dmg just fine.

```bash
$ curl -O https://download.blender.org/release/Blender4.0/blender-4.0.1-macos-arm64.dmg
```

Creates a file ```blender-4.0.1-macos-arm64.dmg``` which is an HTML file.

```bash
$ file blender-4.0.1-macos-arm64.dm

blender-4.0.1-macos-arm64.dmg: HTML document text, ASCII text, with very long lines (6392)
```

I am attaching the output file.
[blender-4.0.1-macos-arm64.dmg.html.gz](https://github.com/Homebrew/discussions/files/13445167/blender-4.0.1-macos-arm64.dmg.html.gz)

I am using `curl` installed with brew, the version 

```bash
$ curl --version
curl 8.4.0 (aarch64-apple-darwin23.0.0) libcurl/8.4.0 (SecureTransport) OpenSSL/3.1.4 zlib/1.2.12 brotli/1.1.0 zstd/1.5.5 libidn2/2.3.4 libssh2/1.11.0 nghttp2/1.58.0 librtmp/2.3 OpenLDAP/2.6.6
Release-Date: 2023-10-11
Protocols: dict file ftp ftps gopher gophers http https imap imaps ldap ldaps mqtt pop3 pop3s rtmp rtsp scp sftp smb smbs smtp smtps telnet tftp
Features: alt-svc AsynchDNS brotli GSS-API HSTS HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz MultiSSL NTLM SPNEGO SSL threadsafe TLS-SRP UnixSockets zstd
```

Strangely, I was able to update other casks just fine, since I have a shell script to auto-install, and update packages and casks. But repeated attempts from the command line on brew do not work.

### Relevant casks

blender

**Comment by DirectorL** [comment]
Not long ago, blender's website was DDoSed, and then they added some restrictions to the website. Brew may have been misjudged as an unfriendly connection. This needs to wait for brew to fix it. It should be fine in a few days. Use the official website first. version

**Reply by SMillerDev** [reply]
What can brew fix about a user agent ban from Blender?

**Reply by DirectorL** [reply]
For example, bypass blender's check, or set more reasonable request headers, or contact blender to add a whitelist. I am not a brewer, and I don't know what they will do, but I know that blender is very commonly used, and this matter will definitely be fixed

**Reply by virusM05** [reply]
https://apple.stackexchange.com/questions/205154/how-to-install-local-dmg-files-via-homebrew
